buildchain/buildchain/salt_tree.py

```diff
@@ -81,158 +81,147 @@ def task__deploy_salt_tree() -> Iterator[types.TaskDict]:
-    Path('salt/metalk8s/kubeadm/init/kubeconfig/lib.sls'),
-    Path('salt/metalk8s/kubeadm/init/kubeconfig/scheduler.sls'),
-
-    Path('salt/metalk8s/kubeadm/init/mark-control-plane/init.sls'),
+    Path('salt/metalk8s/kubernetes/kubelet/files/service-standalone-systemd.conf'),
+    Path('salt/metalk8s/kubernetes/kubelet/files/service-systemd.conf'),
+    Path('salt/metalk8s/kubernetes/kubelet/init.sls'),
+    Path('salt/metalk8s/kubernetes/kubelet/installed.sls'),
+    Path('salt/metalk8s/kubernetes/kubelet/running.sls'),
+    Path('salt/metalk8s/kubernetes/kubelet/standalone.sls'),
+
+    Path('salt/metalk8s/kubernetes/kube-proxy/deployed.sls'),
+
     targets.TemplateFile(
         task_name='configured.sls',
-        source=constants.ROOT/'salt'/'metalk8s'/'kubeadm'/'init'/'mark-control-plane'/'configured.sls.in',
-        destination=constants.ISO_ROOT/'salt'/'metalk8s'/'kubeadm'/'init'/'mark-control-plane'/'configured.sls',
+        # pylint:disable=line-too-long
+        source=constants.ROOT/'salt'/'metalk8s'/'kubernetes'/'mark-control-plane'/'deployed.sls.in',
+        destination=constants.ISO_ROOT/'salt'/'metalk8s'/'kubernetes'/'mark-control-plane'/'deployed.sls',
         context={'VERSION': constants.SHORT_VERSION},
         file_dep=[constants.VERSION_FILE],
     ),
 
-    Path('salt/metalk8s/kubeadm/init/addons/init.sls'),
-    Path('salt/metalk8s/kubeadm/init/addons/kube-proxy.sls'),
-    Path('salt/metalk8s/kubeadm/init/addons/coredns.sls'),
-    Path('salt/metalk8s/kubeadm/init/addons/files/coredns_deployment.yaml.j2'),
+    Path('salt/metalk8s/kubernetes/sa/advertised.sls'),
+    Path('salt/metalk8s/kubernetes/sa/init.sls'),
+    Path('salt/metalk8s/kubernetes/sa/installed.sls'),
 
-    Path('salt/metalk8s/kubelet/init.sls'),
-    Path('salt/metalk8s/kubelet/installed.sls'),
-
-    Path('salt/metalk8s/node/grains.sls'),
+    Path('salt/metalk8s/kubernetes/scheduler/init.sls'),
+    Path('salt/metalk8s/kubernetes/scheduler/installed.sls'),
+    Path('salt/metalk8s/kubernetes/scheduler/kubeconfig.sls'),
 
     Path('salt/metalk8s/macro.sls'),
     Path('salt/metalk8s/map.jinja'),
 
-    Path('salt/metalk8s/python-kubernetes/init.sls'),
-    Path('salt/metalk8s/python-kubernetes/installed.sls'),
-    Path('salt/metalk8s/python-kubernetes/removed.sls'),
+    Path('salt/metalk8s/node/grains.sls'),
+
+    Path('salt/metalk8s/orchestrate/deploy_new_node.sls'),
+
+    Path('salt/metalk8s/registry/files/registry-manifest.yaml.j2'),
+    Path('salt/metalk8s/registry/init.sls'),
+    Path('salt/metalk8s/registry/installed.sls'),
+    Path('salt/metalk8s/registry/macro.sls'),
+    Path('salt/metalk8s/registry/populated.sls'),
 
     Path('salt/metalk8s/repo/configured.sls'),
-    Path('salt/metalk8s/repo/deployed.sls'),
     Path('salt/metalk8s/repo/files/nginx.conf.j2'),
-    Path('salt/metalk8s/repo/files/package-repositories-pod.yaml.j2'),
+    Path('salt/metalk8s/repo/files/package-repositories-manifest.yaml.j2'),
```

Comment:
The other manifests we deploy (`kube-apiserver` etc.) don't have this `-manifest` suffix either ;-)

Reply:
What do you mean ? They have it `control-plane-manifest.yaml` just it's the same for all "control-plane" services so only one file currently in `_internal` directory but as @alexandre-allard-scality suggested I will move it to `kubernetes/files`